bin/entrypoint.sh

```diff
@@ -7,14 +7,14 @@ if [ "$RUN_MODE" = "mock" ]; then
     -p http://127.0.0.1:9999/ \
     -o getlantern \
     -n lantern \
-    -repos getlantern/lantern
+    -repos lantern:getlantern/lantern
 else
   /bin/autoupdate-server \
     -k /keys/private.key \
     -l 0.0.0.0:9999 \
     -p https://update.getlantern.org/ \
     -o getlantern \
-    -r 0.3 \
+    -r 1000 \
```

Comment:
Drive-by. Replace the rollout rate with rate limit. It's long been planed but I could not find the ticket.